tests/rkt_list_test.go

```diff
@@ -57,9 +57,6 @@ func TestRktList(t *testing.T) {
 	// Get hash
 	imageID := fmt.Sprintf("sha512-%s", imgID.hash[:12])
 
-	tmpDir := createTempDirOrPanic(imgName)
-	defer os.RemoveAll(tmpDir)
```

Comment:
Good cleanup of unused directory. But not sure it belongs to this PR.

Also, same cleanup to do line 164 in `TestRktListCreatedStarted`.